management/virtualmachine/resourceextensions_test.go

```diff
@@ -0,0 +1,38 @@
+package virtualmachine
+
+import (
+	"flag"
+	"testing"
+
+	"github.com/MSOpenTech/azure-sdk-for-go/management/testutils"
+)
+
+func init() {
+	flag.BoolVar(&extraVerbose, "resourceextensions.verbose", false, "List all resource extensions")

```

Comment:
@paulmey usually tests are ran without `-test.v`, and when it fails, all t.Log output is also printed (even without `-test.v`). That's why nobody probably needs an extra level of verbosity.